### PR TITLE
Fix dag sort in dag stats

### PR DESCRIPTION
**Pull request by ziliangpeng**
Fix a bug in stat printer in dag_processing to correctly sort dag list by last runtime.

### CODE REVIEW

**Comment by dimberman** [comment]
@ziliangpeng can you add a test?

**Comment by uranusjr** [comment]
This may be a good chance to refactor the entire block to use namedtuple as well, to prevent future breakage.

File: airflow/utils/dag_processing.py [hunk 1]
```diff
@@ -834,7 +834,7 @@ def _log_file_processing_stats(self, known_file_paths):
             rows.append((file_path, processor_pid, runtime, num_dags, num_errors, last_runtime, last_run))
 
         # Sort by longest last runtime. (Can't sort None values in python3)
```
Comment (jhtimmins) [inline airflow/utils/dag_processing.py:836]: This seems to suggest that it's sorting the results by the length of each DAG's last runtime. Is your change sorting by the time of the last runtime? Or was it not sorting in the way the comment suggests and you're just fixing it?

If it's the former, can you update the comment to reflect the new sort order?

Comment (ziliangpeng) [reply airflow/utils/dag_processing.py:836]: The comment and the code was not consistent, as you can see from the code above
```
rows.append((file_path, processor_pid, runtime, num_dags, num_errors, last_runtime, last_run))
```
index (5) is last_runtime. I'm just making the code and comment consistent.

It was probably that when more fields were added into the row, and sort field was not updated.

**Comment by ziliangpeng** [comment]
@dimberman @uranusjr It doesn't seem easy to add a unit test without some major refactoring of the code structure, which expanded the scope of the PR. 

Can adding unit test be a different task, and achieved by another PR?

**Comment by uranusjr** [comment]
If you can refactor this to something like

```python
Row = namedtuple("Row", "file_path processor_pid runtime num_dags num_errors last_runtime last_run")

rows = []
for file_path in known_file_paths:
    ...
    row = Row(
        file_path=file_path,
        processor_pid=processor_pid,
        runtime=runtime,
        num_dags=num_dags,
        num_errors=num_errors,
        last_runtime=last_runtime,
        last_run=last_run,
    )
    rows.append(row)
rows = sorted(rows, lambda row: row.last_runtime or 0.0)
```

Then I think this is fine without a unit test (since the logic is obvious enough).

**Pull request closed without merging**